shotover-proxy/tests/opensearch_int_tests/mod.rs

```diff
@@ -87,3 +69,183 @@ async fn passthrough_standard() {
         10
     );
 }
+
+// TODO this test fails even if we are waiting for the health status to go green
+// async fn test_health(client: &OpenSearch) {
```

Comment:
If this is a test case we want to get working eventually, lets not leave the code commented out. (otherwise it'll bit rot)
And instead just remove the failing assert and include a `TODO: add assert that X once Y functionality is fixed/implemented`

If this test case was a misstep and its not possible to get working then just delete it entirely.